[2024-12-26T14:33:00.279+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-26T14:33:00.318+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_python_operator.virtualenv_python manual__2024-12-26T14:32:47.718332+00:00 [queued]>
[2024-12-26T14:33:00.334+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_python_operator.virtualenv_python manual__2024-12-26T14:32:47.718332+00:00 [queued]>
[2024-12-26T14:33:00.334+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-26T14:33:00.354+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonVirtualenvOperator): virtualenv_python> on 2024-12-26 14:32:47.718332+00:00
[2024-12-26T14:33:00.376+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1135) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-26T14:33:00.377+0000] {standard_task_runner.py:72} INFO - Started process 1136 to run task
[2024-12-26T14:33:00.376+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'example_python_operator', 'virtualenv_python', 'manual__2024-12-26T14:32:47.718332+00:00', '--job-id', '21', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_python_operator.py', '--cfg-path', '/tmp/tmpho3pk8li']
[2024-12-26T14:33:00.378+0000] {standard_task_runner.py:105} INFO - Job 21: Subtask virtualenv_python
[2024-12-26T14:33:00.476+0000] {task_command.py:467} INFO - Running <TaskInstance: example_python_operator.virtualenv_python manual__2024-12-26T14:32:47.718332+00:00 [running]> on host ebf312bc13e9
[2024-12-26T14:33:00.631+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_python_operator' AIRFLOW_CTX_TASK_ID='virtualenv_python' AIRFLOW_CTX_EXECUTION_DATE='2024-12-26T14:32:47.718332+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-26T14:32:47.718332+00:00'
[2024-12-26T14:33:00.633+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-26T14:33:00.633+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-26T14:33:00.633+0000] {logging_mixin.py:190} INFO - Current task name:virtualenv_python state:running start_date:2024-12-26 14:33:00.319385+00:00
[2024-12-26T14:33:00.634+0000] {logging_mixin.py:190} INFO - Dag name:example_python_operator and current dag run status:running
[2024-12-26T14:33:00.634+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-26T14:33:00.634+0000] {baseoperator.py:421} WARNING - PythonVirtualenvOperator.execute cannot be called outside TaskInstance!
[2024-12-26T14:33:00.636+0000] {process_utils.py:186} INFO - Executing cmd: /home/***/.local/bin/python -m virtualenv /tmp/venvyz2xtj9t --python=python
[2024-12-26T14:33:00.637+0000] {process_utils.py:190} INFO - Output:
[2024-12-26T14:33:01.903+0000] {process_utils.py:194} INFO - created virtual environment CPython3.12.8.final.0-64 in 647ms
[2024-12-26T14:33:01.904+0000] {process_utils.py:194} INFO -   creator CPython3Posix(dest=/tmp/venvyz2xtj9t, clear=False, no_vcs_ignore=False, global=False)
[2024-12-26T14:33:01.905+0000] {process_utils.py:194} INFO -   seeder FromAppData(download=False, pip=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2024-12-26T14:33:01.906+0000] {process_utils.py:194} INFO -     added seed packages: pip==24.3.1
[2024-12-26T14:33:01.906+0000] {process_utils.py:194} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2024-12-26T14:33:01.935+0000] {process_utils.py:186} INFO - Executing cmd: /tmp/venvyz2xtj9t/bin/pip install -r /tmp/venvyz2xtj9t/requirements.txt
[2024-12-26T14:33:01.937+0000] {process_utils.py:190} INFO - Output:
[2024-12-26T14:33:03.159+0000] {process_utils.py:194} INFO - WARNING: The directory '/home/***/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you should use sudo's -H flag.
[2024-12-26T14:33:03.568+0000] {process_utils.py:194} INFO - Collecting colorama==0.4.0 (from -r /tmp/venvyz2xtj9t/requirements.txt (line 1))
[2024-12-26T14:33:03.770+0000] {process_utils.py:194} INFO -   Downloading colorama-0.4.0-py2.py3-none-any.whl.metadata (13 kB)
[2024-12-26T14:33:03.809+0000] {process_utils.py:194} INFO - Downloading colorama-0.4.0-py2.py3-none-any.whl (21 kB)
[2024-12-26T14:33:03.845+0000] {process_utils.py:194} INFO - Installing collected packages: colorama
[2024-12-26T14:33:03.866+0000] {process_utils.py:194} INFO - Successfully installed colorama-0.4.0
[2024-12-26T14:33:04.226+0000] {process_utils.py:186} INFO - Executing cmd: /tmp/venvyz2xtj9t/bin/python /tmp/venv-callkzuqsdh8/script.py /tmp/venv-callkzuqsdh8/script.in /tmp/venv-callkzuqsdh8/script.out /tmp/venv-callkzuqsdh8/string_args.txt /tmp/venv-callkzuqsdh8/termination.log
[2024-12-26T14:33:04.227+0000] {process_utils.py:190} INFO - Output:
[2024-12-26T14:33:04.338+0000] {process_utils.py:194} INFO - [31msome red text
[2024-12-26T14:33:04.340+0000] {process_utils.py:194} INFO - [42mand with a green background
[2024-12-26T14:33:04.340+0000] {process_utils.py:194} INFO - [2mand in dim text
[2024-12-26T14:33:04.341+0000] {process_utils.py:194} INFO - [0m
[2024-12-26T14:33:04.341+0000] {process_utils.py:194} INFO - [2mPlease wait...
[2024-12-26T14:33:05.339+0000] {process_utils.py:194} INFO - [2mPlease wait...
[2024-12-26T14:33:06.339+0000] {process_utils.py:194} INFO - [2mPlease wait...
[2024-12-26T14:33:07.339+0000] {process_utils.py:194} INFO - [2mPlease wait...
[2024-12-26T14:33:08.339+0000] {process_utils.py:194} INFO - Finished
[2024-12-26T14:33:08.393+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-26T14:33:08.415+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-26T14:33:08.416+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=example_python_operator, task_id=virtualenv_python, run_id=manual__2024-12-26T14:32:47.718332+00:00, execution_date=20241226T143247, start_date=20241226T143300, end_date=20241226T143308
[2024-12-26T14:33:08.435+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-26T14:33:08.435+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-26T14:33:08.436+0000] {logging_mixin.py:190} INFO - Dag name:example_python_operator queued_at:2024-12-26 14:32:47.745503+00:00
[2024-12-26T14:33:08.436+0000] {logging_mixin.py:190} INFO - Task hostname:ebf312bc13e9 operator:PythonVirtualenvOperator
[2024-12-26T14:33:08.486+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-26T14:33:08.534+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-26T14:33:08.537+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
